packages/controllers/dataset_detail.coffee

```diff
@@ -0,0 +1,17 @@
+if Meteor.isClient
+  Template.datasetDetail.onCreated ->
+    @subscribe('datasetDetail', @data.datasetId)
+
+  Template.datasetDetail.helpers
+    dataset: ->
+      Datasets.findOne(@datasetId)
+
+if Meteor.isServer
+  Meteor.publish 'datasetDetail', (id) ->
+    dataset = Datasets.findOne
+      _id: id
+      createdById: @userId
+    if dataset
+      [Datasets.find(dataset._id), RawFiles.find(dataset.file)]
```

Comment:
Could you write a method on the Dataset model that returns the equivalent of RawFiles.find(dataset.file)? This way the controller doesn't need to know what RawFiles are, it can just trust that the model is giving it the right object (in this case a cursor).